sycl/source/detail/program_manager/program_manager.cpp

```diff
@@ -367,11 +367,36 @@ RT::PiProgram ProgramManager::getBuiltPIProgram(OSModuleHandle M,
   auto GetF = [](const Locked<ProgramCacheT> &LockedCache) -> ProgramCacheT & {
     return LockedCache.get();
   };
-  auto BuildF = [this, &M, &KSId, &Context, &Device, Prg,
-                 &JITCompilationIsRequired] {
-    const RTDeviceBinaryImage &Img =
-        getDeviceImage(M, KSId, Context, Device, JITCompilationIsRequired);
+  string_class BuildOptions;
```

Comment:
Here and in few other places in the patch - SYCL 2020 deprecates xxx_class aliases
```suggestion
  std::string BuildOptions;
```